Admin css overhaul pt2

gingervitis opened the pull request:
This overhaul work includes:

- I took out Ant Dark style sheet. People seem to like dark mode, so we're only going to be dark and not have to bother with 2 different stylings.
- I overwrote a bunch Ant's component styles to match Owncast's dark/purple  themed.
- Cleaned up some layout styling for a few pages
- ToggleSwitch and TextField components share the same stylesheet so their layout can be consistent in a single form


Still TODO:
- Clean up css var usage for consistency
- Clean up Types management when using non-submit ToggleSwitch
- Finish cleaning up styles on Slider components.
- Finish up fixing layout on Video Variant modal 
- Fix styles on other non-Config pages
- Colors look too dark on some screens and areas, things may be hard to read.  I'll adjust.
- Write up some Tips on easy styling things to do or components to use when you want to add your own forms

nebunez reviewed (commented):
Is there a color palette mockup anywhere? If not, I'd suggest one--there are many tools with which to do so. This will help organize the colors in context outside of the code.
Also, I recommend the WAVE (Wave Accessibility Evaluation Tool) browser plugin. There should be a Chrome version as well as Firefox. This tool will evaluate a page as you look at it and give accessibility reports as well as tools to fix issues.

gingervitis commented:
> Is there a color palette mockup anywhere? If not, I'd suggest one--there are many tools with which to do so. This will help organize the colors in context outside of the code.
> Also, I recommend the WAVE (Wave Accessibility Evaluation Tool) browser plugin. There should be a Chrome version as well as Firefox. This tool will evaluate a page as you look at it and give accessibility reports as well as tools to fix issues.

Perhaps it is about time I made and settle on a REAL Owncast color palette across all the Owncast apps.  Since there was never any formal design mock, I've just been randomly winging it so far, spending too many minutes tweaking color pickers until things look ok in some localized page setting.

The CSS variables file I've started will be a good start, that I need to clean up and consistently use around the stylesheets.

I also thought about installing [Storybook](https://storybook.js.org) here, as (maybe a too-heavy-for-this-project) dev tool that can serve as a small show case of colors and components.

nebunez commented:
> I also thought about installing [Storybook](https://storybook.js.org) here, as (maybe a too-heavy-for-this-project) dev tool that can serve as a small show case of colors and components.

I have seen Storybook before, and it is a great tool, but I agree it's probably too much for this project--especially since we're already using a (modified) design library that has component examples.

gingervitis commented on components/config/form-toggleswitch-with-submit.tsx:
```diff
@@ -42,41 +56,52 @@ export default function ToggleSwitch(props: ToggleSwitchProps) {
+      });
+      resetTimer = setTimeout(resetStates, RESET_TIMEOUT);
+    }
+    if (onChange) {
```
I defaulted the onChange to `null` instead of `()=>{}`

gingervitis commented:
Thank you for helping to review @nebunez! I addressed your comments and I think I will merge what I have so far in here into 0.0.6. I'll address the TODO items  and accessible color updates in a new branch.
